jupyter-extension/Dockerfile

```diff
@@ -31,4 +31,9 @@ USER $NB_UID
 COPY dist dist
 WORKDIR /home/jovyan
 RUN pip install `find /app/dist/ -name \*.whl` nbgitpuller
-RUN pip install determined==0.23.3
\ No newline at end of file
+RUN pip install determined==0.23.3
+RUN pip install jupyterlab_pachyderm
```

Comment:
I don't think you want this. This command will install the latest published jupyterlab_pachyderm package and not the local one. The `pip install ``find /app/dist/ -name \*.whl\`` ngitpuller` above should be installing the local wheel from the package, assuming you've built it.

Reply:
It seems that the docker image does not work if I do not install the jupyterlab_pachyderm package, even though the wheel files you mention are there and packages are getting installed. I can remove it if the jupyterlab_pachyderm package is installed from the wheel file.

Reply:
I have removed the entry `RUN pip install jupyterlab_pachyderm`